README.md

```diff
@@ -22,10 +22,10 @@ pod 'Adhan'
 
 ### Carthage
 
-For [Carthage](https://github.com/Carthage/Carthage) add the following to your [Cartfile](https://github.com/Carthage/Carthage/blob/master/Documentation/Artifacts.md#cartfile):
+For [Carthage](https://github.com/Carthage/Carthage) add the following to your [Cartfile](https://github.com/Carthage/Carthage/blob/main/Documentation/Artifacts.md#cartfile):
```

Comment:
Carthage default branch is still `master`

Reply:
thanks for catching it. Resolved.